test/framework/k8s_cluster.go

```diff
@@ -312,39 +235,92 @@ func (c *K8sCluster) DeployKuma(mode ...string) error {
 		DefaultRetries,
 		DefaultTimeout)
 
-	_, err = retry.DoWithRetryE(c.t,
-		"get default mesh",
-		DefaultRetries,
-		DefaultTimeout,
-		func() (s string, err error) {
-			return k8s.RunKubectlAndGetOutputE(c.t, c.GetKubectlOptions(), "get", "mesh", "default")
-		})
+	//_, err = retry.DoWithRetryE(c.t,
+	//	"get default mesh",
+	//	DefaultRetries,
+	//	DefaultTimeout,
+	//	func() (s string, err error) {
+	//		return k8s.RunKubectlAndGetOutputE(c.t, c.GetKubectlOptions(), "get", "mesh", "default")
+	//	})
+	//if err != nil {
+	//	return nil, err
+	//}
```

Comment:
Do we need this commented code?

Reply:
Not sure, wanted to have @lobkovilya's opinion why he added it in his Ingress work. I think we should not enforce getting the default mesh as part of the Kuma deployment. In Global mode, you can very well not have "default" mesh at all.

Reply:
Could you please just place it into block:
```go
if len(mode) == 0 || mode[0] != core.Global {
...
}
```
I had flaky test locally, it failed to deploy example-app right after deploying KumaCP. That retry helped

Reply:
But we have VerifyKuma, which more or less does the same?

Reply:
From what I see VerifyKuma doesn't check default mesh. But okay, let's delete it and observe, we always can restore it in case of flakiness